src/hamiltonian.F90

```diff
@@ -657,6 +657,23 @@ subroutine hamiltonian_write_rmn()
     end do
     
     close(file_unit)
+!  
+!   RM and SP - write to file vectors b and their weight wb for each k-point 
+!   This is used by EPW to compute the velocity. 
+!   Note that you need to put in your Wannier input:
+!     write_hr        = .true.
+!     write_rmn       = .true.
+!     write_bvec      = .true. 
+    if (write_bvec) then
+      open(file_unit,file=trim(seedname)//'.bvec',form='formatted',status='unknown',err=101)
+      write(file_unit,*) nntot
+      do nkp = 1, num_kpts
+         do nn = 1, nntot
+            write (file_unit,'(4F12.6)') bk(:,nn,nkp), wb(nn)
```

Comment:
Another request about the format of the file: could you add two header lines, as in most of the other files:
1. first line as a comment, including a timestamp, in order to ease debug in case people regenerate many times the file and want to know if it was regenerated or not
2. Second line with the size of the matrix (two integers, `num_kpts` and `nntot`)

You can give a look to other files to see how the header looks like